src/plugin/ws.ts

```diff
@@ -84,4 +84,39 @@ export class WsSignerPlugin implements SignerPlugin {
       };
     });
   }
+
+  public sendOriginInfo(method: string, plugin: string = ""): Promise<string> {
```

Comment:
do not create new interfaces. use PluginOpts

Reply:
updated